base_location_code_program_territory/models/res_city_zip.py

```diff
@@ -24,7 +24,8 @@ class ResCityZip(models.Model):
     code_program_territory_id = fields.Many2one(
         string="Code Program Territory",
         comodel_name="res.partner.code.program.territory",
-        domain="['|', ('code_program_id', '=?', code_program_id), ('code_program_id', '=', False)]",
+        domain="['|', ('code_program_id', '=?', code_program_id),\
```

Comment:
Could you please tell me why the "=?" operator is used instead of "="?

Reply:
No, :(

I just added a space for the pre-commit